rmgpy/molecule/resonance.py

```diff
@@ -827,7 +828,7 @@ def generate_kekule_structure(mol):
     cython.declare(atom=Vertex, molecule=Graph)
 
     for atom in mol.atoms:
-        if type(atom) != type(Atom()):
+        if not isinstance(atom,CuttingLabel):
```

Comment:
I think it should be just plain `isinstance` instead of `not isinstance`? i.e. we should skip the atom if it is a cutting label